adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralRewardedAdRenderer.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+
+@interface GADMAdapterMintegralRewardedAdRenderer : NSObject
+
+/// Asks the receiver to load the ad configuration.
+- (void)renderRewardedAdForAdConfiguration:(nonnull GADMediationRewardedAdConfiguration *)adConfiguration
```

Comment:
Rename this to `loadRewardedAdForAdConfiguration`.

Reply:
ok,I have updated.